cogs/staff_mod.py

```diff
@@ -82,6 +82,7 @@ async def case(self, ctx, number: int):
         await ctx.send(embed=embed)
 
     @commands.command()
+    @commands.has_permissions(kick_members=True)
```

Comment:
Following the syntax of the other commands, please make this above the commands decorator

Reply:
And while you're at it, will you are at it, will you add a permissions decorator, with manage_messages to the purge and case commands, pls thx

Reply:
Done